vue/components/ui/molecules/form-input/form-input.vue

```diff
@@ -1,28 +1,43 @@
 <template>
-    <div class="form-input">
-        <label-ripe class="header" v-bind:text="header" v-bind:for="id" v-if="header" />
-        <div class="content">
-            <slot />
-        </div>
+    <div class="form-input" v-bind:class="alignment">

```

Comment:
Move `alignment` to a `classes` `computed`, similarly to what we have in other components.